agent-core/src/main/java/com/newrelic/agent/android/harvest/HarvestTimer.java

```diff
@@ -145,20 +141,47 @@ public void shutdown() {
         scheduler.shutdownNow();
     }
 
-    // Runs a tick of the Harvester immediately, disregarding any 'time since last tick' limits.
-    public void tickNow() {
-        final HarvestTimer timer = this;
-        ScheduledFuture future = scheduler.schedule(new Runnable() {
-            @Override
-            public void run() {
-                timer.tick();
-            }
-        }, 0, TimeUnit.SECONDS);
+    public void restart() {
```

Comment:
If this method is not used, go ahead and remove it. It was more for reference of what is required to restart the HarvestTimer.